src/App.tsx

```diff
@@ -1,24 +1,238 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useMemo, useState } from 'react';
+import {
+  addTodo,
+  deleteTodo,
+  getTodos,
+  updateTodoStatus,
+  updateTodoTitle,
+} from './api/todos';
+import { Footer } from './components/Footer';
+import { Header } from './components/Header';
+import { Notification } from './components/Notification';
+import { TodoList } from './components/TodoList';
+import { UserWarning } from './components/UserWarning/UserWarning';
+import { ErrorMessages } from './types/ErrorMessages';
+import { FilterBy } from './types/FilterBy';
+import { Todo } from './types/Todo';
+import { closeNotification } from './utils/closeNotification';
+import { prepareTodo } from './utils/prepareTodo';
 
-const USER_ID = 0;
+const USER_ID = 6337;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterBy, setFilterBy] = useState<FilterBy>(FilterBy.ALL);
+  const [title, setTitle] = useState('');
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [isError, setIsError] = useState(false);
+  const [isInputDisabled, setIsInputDisabled] = useState(false);
+  const [
+    errorMessage,
+    setErrorMessage,
+  ] = useState<ErrorMessages>(ErrorMessages.NONE);
+  const [isUpdatingTodoId, setIsUpdatingTodoId] = useState(0);
+
+  const activeTodos = todos
+    .filter(todo => !todo.completed);
+  const hasActiveTodos = activeTodos.length > 0;
+  const howManyActiveTodosLeft = activeTodos.length;
+  const hasCompletedTodos = todos.some(todo => todo.completed);
+
+  const loadTodos = async () => {
+    try {
+      const loadedTodos = await getTodos(USER_ID);
+
+      setTodos(loadedTodos);
+    } catch (error) {
+      setIsError(true);
+      closeNotification(setIsError, false, 3000);
```

Comment:
It seems to me, here it also needs to add `setErrorMessage()` something like "Unable to load todos".